modules/ROOT/pages/_partials/block-authors-notes.adoc

```diff
@@ -39,7 +39,7 @@ ifeval::["{do-it}" == "yes"]
 ====
 CONTENT UNDER DEVELOPMENT
```

Comment:
General: Not sure of the purpose of the "Content Under Development" tag. Not sure if its common across server etc. Seems very odd to have that in official non-beta docs. Please clean up. Docs will always evolve and technically will always remain under development

Reply:
Content in _partials is used in includes. This is just a utility 'page' containing blocks to be used on a page as and when needed; a macro if you like that helps ensure consistency of presentation.